src/style.css

```diff
@@ -0,0 +1,145 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
```

Comment:
make it `card` bem block and create a separate file
<img width="510" alt="image" src="https://user-images.githubusercontent.com/102411130/223655916-444b7fbe-0663-4f31-97e9-8fb8a84a8733.png">

Reply:
not fixed